requirements.in

```diff
@@ -1,5 +1,5 @@
 argcomplete
-boto3
+boto3>=1.16.46
```

Comment:
@mcrowson this (or 1.16.56) was the first version with the docker image support, but you'll see that `pip-compile --output-file=requirements.txt requirements.in`  generates a `requirements.txt` with `boto3==1.17.28`

Reply:
I think some other library we use must require >1.17.28. If that is the case, our requirements version should reflect that as well. The 1.16.48 is just not accurate to what would get installed if that is the case.

Reply:
updated